### ISSUE TITLE
[FEATURE] - Enable ability to navigate between files by default

### ISSUE THREAD

**Issue by mikey-t** [alefragnani/vscode-numbered-bookmarks#165]
There is a similar existing closed issue, but my suggestion is to adjust the feature rather than use the workaround (change the setting): https://github.com/alefragnani/vscode-numbered-bookmarks/issues/155

**Feature Idea**

Instead of requiring the changing of the setting "numberedBookmarks.navigateThroughAllFiles" to "allowDuplicates", it would be really nice to have navigation between bookmarks in different files work by default. Said another way: if necessary, disable the ability to have duplicates across files by default so that jumping between files works without the need to change settings.

**Other Info**

I feel like the most common scenario is where there's bookmark 1 in file A and bookmark 2 in file B and I just want to jump between them. By default this doesn't work. The "allowDuplicates" setting suggests that I'm trying to have bookmark 1 and 2 in file A and bookmark 1 and 2 in file B also, which has nothing to do with the original issue and is unintuitive for most people that work in projects with many smaller files rather than jumping between places in bigger files.

I really like the way Jetbrains does this where you're not allowed to have duplciates and the numbers are solution wide. You get a little popup if you try and create a new numbered bookmark for a number that is already taken. The popup asks if you want to overwrite, and has a handy checkbox to not ask again for people who frequently create and overwrite bookmarks and don't need/want to be warned (which seems like the most common thing for using numbered bookmarks vs longer lists of customizable/nameable bookmarks). For this numbere bookmarks vscode plugin, the popup is not really that helpful. I feel the default functionality should be to simply do what the user requests when they set a numbered bookmark (overwriting the previous bookmark) and provide the setting to change this for the people that actually want duplicates.

Maybe too much information at this point, but I feel like people that are going to have lots of duplicates are going to have too many bookmarks for the numbered system to really give them the most benefit - they would probably instead benefit most from your other vscode bookmarks extension where they can just have a big list of all the bookmarks and organize and name them however they want. But for most of us using this numbered bookmarks extension, we just want a quick way to keep a stack in our head of a couple of numbered bookmarks in different files that we can quickly swap between and frequently change and overwrite.

**Comment by alefragnani** [comment]
Hi @mikey-t ,

As explained in #6 (where the feature was originally requested), this extension was made to be _in Delphi style_, which was my main IDE for most of my development days, and I wanted to have the exact same behavior in VS Code. But, based on user comments, I noticed the changes could improve their use, and decided to improve its behavior to support other scenarios.  And that’s how `numberedBookmarks.navigateThroughAllFiles` setting was created (https://github.com/alefragnani/vscode-numbered-bookmarks/issues/6#issuecomment-296209464).

So, in the end, the feature that added the `numberedBookmarks.navigateThroughAllFiles` is not a _workaround_ for the `replace` or `allowDuplicates` options, but an improvement to the original behavior. 

On the other hand, changing the _default_ behavior to be other than the original (Delphi like) one was not an option, basically for two reasons:
1. The extension was created to mimic Delphi, and remains with this in mind
2. Changing the default behavior could affect the entire user base (today, around 180k) that didn’t touch the setting

If the setting behavior is not clear, maybe with #117 launch this could be improved. Adding a step to _”configure Numbered Bookmarks if you came from IntelliJ IDEs”_, or something like that could help those that came from this IDE.

BTW, I started using IntelliJ IDEs a while ago (Rider to be more precise) and its mnemonic bookmarks is sure interesting. A bit complicated at some points, but still, interesting.

Hope this helps
